test/resources/knativeserving.go

```diff
@@ -65,14 +71,18 @@ func WaitForKnativeServingState(clients servingv1alpha1.KnativeServingInterface,
 
 // CreateKnativeServing creates a KnativeServing with the name names.KnativeServing under the namespace names.Namespace.
 func CreateKnativeServing(clients servingv1alpha1.KnativeServingInterface, names test.ResourceNames) (*v1alpha1.KnativeServing, error) {
```

Comment:
maybe rename this to EnsureKnativeServingExists?

Reply:
Done.